### PR TITLE
Feat/tooltip

### PR DESCRIPTION
**Pull request by dpellier**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-storybook#521.0 ✘  
* 
  * build ✘

File: packages/libraries/core/src/components/tooltip/tooltip/ods-tooltip-controller.spec.ts [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+      it('should do nothing if there is no surface', () => {
+        expect(() => {
+          controller.checkForClickOutside({ target });
+        }).not.toThrow();
+        expect(component.surface).toBeUndefined();
+      });
+
+      it('should do nothing if surface is not opened', () => {
+        component.surface = new OcdkSurfaceMock() as unknown as OcdkSurface;
```
Comment (ady642) [inline packages/libraries/core/src/components/tooltip/tooltip/ods-tooltip-controller.spec.ts:60]: we can't cast directly in OcdkSurface ?

**Comment by ovh-cds** [comment]
CDS Report build-storybook#521.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#522.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#524.0 ✘  
* 
  * build ✘

File: packages/libraries/core/src/components/tooltip/tooltip/ods-tooltip-controller.spec.ts (hunk 1 above)
Comment (dpellier) [reply packages/libraries/core/src/components/tooltip/tooltip/ods-tooltip-controller.spec.ts:60]: `neither type sufficiently overlaps with the other`
Though it seems to be doable as it's done on popover `https://github.com/ovh/design-system/pull/14/files#diff-c2485612538bedda38b90d6226829f889891f5cc62e1d713da2685976f0c0a56` and the setup is quite similar

Any idea what may be missing @skhamvon @astagnol ?

**Comment by ovh-cds** [comment]
CDS Report build-storybook#528.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#531.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#532.0 ✘  
* 
  * build ✘

**Review by aesteves60 — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#779.0 ✘  
* 
  * build ✘

**Review by astagnol — APPROVED** [review]

**Pull request merged**